readme.md

```diff
@@ -154,8 +154,7 @@ cd bakerydemo
 pip install -r requirements/development.txt
 ```
 
-Next, we'll set up our local environment variables. We use [django-dotenv](https://github.com/jpadilla/django-dotenv)
-to help with this. It reads environment variables located in a file name `.env` in the top level directory of the project. The only variable we need to start is `DJANGO_SETTINGS_MODULE`:
+Next, we need to create the files `.env` and `bakerydemo/settings/local.py`. The `.env` file is used by [django-dotenv](https://github.com/jpadilla/django-dotenv) to store environment variables relevant to the local installation, such as `DJANGO_SETTINGS_MODULE`. Similarly, `bakerydemo/settings/local.py` can be used to store local Django settings such as database connection details that need to be kept outside of version control. Both of these files are initially empty, as the bakerydemo project provides usable defaults for all settings, but they must exist for the server to start:
```

Comment:
> but they must exist for the server to start

I think the server would still start without `.env`, but `django-dotenv` will print a warning in the console. We use `try`/`except` when importing `local.py` (e831dcee2790c22c597745b89b9bb28ec2edaa63), so it's not required too, but I still think it's worthwhile to keep the instructions for adding these files so they don't have to figure this out later.

Maybe "but they must exist for the server to start gracefully" would be more appropriate?